src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
+    <appender name="default" class="ch.qos.logback.core.FileAppender">
+        <file>logs/console.log</file>
+        <encoder>
+            <pattern>%date %level [%thread] %logger{64} %msg%n</pattern>
```

Comment:
I prefer a file appender to eliminate noisy logging outputs in terminal when running tests